src/lib/options.js

```diff
@@ -66,8 +66,8 @@ function defaultOptions() {
         baseUriPath: process.env.BASE_URI_PATH || '',
         unleashUrl: process.env.UNLEASH_URL || 'http://localhost:4242',
         serverMetrics: true,
-        enableLegacyRoutes: false,
-        extendedPermissions: false,
+        enableLegacyRoutes: false, // deprecated. Remove in v4,
+        extendedPermissions: false, // deprecated. Remove in v4,
```

Comment:
I added a few deprecation marks here to remind us to remove these as part of v4.